scripts/build_android.bat

```diff
@@ -0,0 +1,80 @@
+@echo off
+setlocal
+
+:: =============================
+:: Configuration
+:: =============================
+set "APP_NAME=%~1"
+if "%APP_NAME%"=="" (
+    echo [ERROR] Application name argument missing.
+    exit /b 1
+)
+
+set "QT_ROOT=C:\Qt6.8.2\6.8.2"
+set "QT_ANDROID_BIN=%QT_ROOT%\android_arm64_v8a\bin"
+set "QT_DEPLOY_TOOL=%QT_ROOT%\mingw_64\bin\androiddeployqt.exe"
+set "MAKE_EXE=%USERPROFILE%\AppData\Local\Android\Sdk\ndk\28.0.12433566\prebuilt\windows-x86_64\bin\make.exe"
+set "ANDROID_NDK_ROOT=%USERPROFILE%\AppData\Local\Android\Sdk\ndk\28.0.12433566"
+set "ANDROID_PLATFORM=android-34"
+
+:: =============================
+:: Workspace & Paths
+:: =============================
+cd /d "%~dp0\.." || exit /b 1
+set "WORKSPACE=%cd%"
+set "BUILD_DIR=%WORKSPACE%\build"
+set "ANDROID_BUILD_DIR=%BUILD_DIR%\android-build"
+set "PRO_FILE=%WORKSPACE%\%APP_NAME%\%APP_NAME%.pro"
+set "DEPLOY_JSON=%BUILD_DIR%\android-DSA_%APP_NAME%_Qt-deployment-settings.json"
+
+:: =============================
+:: Prepare Build Directory
+:: =============================
+echo [INFO] Cleaning build directory...
+if exist "%BUILD_DIR%" rmdir /S /Q "%BUILD_DIR%"
+mkdir "%BUILD_DIR%"
+cd "%BUILD_DIR%" || exit /b 1
+
+:: =============================
+:: Run QMake and Make
+:: =============================
+echo [INFO] Running qmake...
+call "%QT_ANDROID_BIN%\qmake.bat" ^
+    "%PRO_FILE%" ^
+    -spec android-clang ^
+    CONFIG+=debug CONFIG+=qml_debug || exit /b 1
```

Comment:
This will create a debug build. That simplifies things if you don't want to deal with codesigning, but it will be problematic if anyone is checking performance. I recommend having both options in here, and the release build mode should be the default. If that is too much scope creep, than only the release build should suffice.

Reply:
If a debug build is expected, that sounds fine. Presumably someone will eventually want to run a release build, so that should be an option but for now this seems sufficient.